backend/contracts/src/main/java/org/egov/works/enrichment/ContractEnrichment.java

```diff
@@ -284,6 +330,17 @@ private void addLineItem(List<LineItems> refinedLineItems,LineItems lineItem,Lis
 
     private void enrichIdsAgreementDateAndAuditDetailsOnCreate(ContractRequest contractRequest) {
         Contract contract = contractRequest.getContract();
+        if (contract.getBusinessService() != null && contract.getBusinessService().equalsIgnoreCase(WORKFLOW_BUSINESS_SERVICE)) {
+            contract.setOldUuid(contract.getId());
```

Comment:
we can not set oldUuid, versionNumber directly from request, first get the active contract from db and update the request

Reply:
done